lib/SPIRV/libSPIRV/SPIRVType.cpp

```diff
@@ -183,7 +184,9 @@ bool SPIRVType::isTypeInt(unsigned Bits) const {
   return isType<SPIRVTypeInt>(this, Bits);
 }
 
-bool SPIRVType::isTypePointer() const { return OpCode == OpTypePointer; }
+bool SPIRVType::isTypePointer() const {
```

Comment:
Looking at the specification it might be useful to also introduce `isTypeUntypedPointer` method (new instructions have the following restriction: Result Type must be an OpTypeUntypedPointerKHR). Not sure if it should go in this or next PR.

Reply:
I'll add it in a untyped variable PR, if it won't be needed in this one.